content/en/news/releases/1.9.x/announcing-1.9.1/index.md

```diff
@@ -14,19 +14,11 @@ as well as bug fixes to improve robustness.
 
 This release note describes what’s different between Istio 1.9.0 and Istio 1.9.1.
 
-{{< warning >}}
```

Comment:
@duderino mentioned we shouldn't remove this in case people had already looked up this page and then revisited it, but to add something to it around "Qual testing has passed."

I'm fine with removing it, but I'd check with Josh.

Reply:
I'm OK either way.

Reply:
My feeling is, there will be 1% of the total people who will view this page, have seen it already, and the other 99% who will look from now on, will be confused by the notice.

I'll add a mention inline.